### ISSUE TITLE
[Adaptive Cards> WeatherLarge.Json]: In normal mode, Tooltip of Days name is getting cropped. 

### ISSUE THREAD

**Issue by vagpt** [microsoft/AdaptiveCards#8981]
### Target Platforms

UWP

### SDK Version

1.0.1.0

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
Application: Adaptive Cards Universal
OS Build: 24H2 (OS Build 26252.5000)
App Version: 1.0.1.0

**Repro Steps:**
1. Open the Adaptive Cards Universal application.
2. Adaptive Card UWP Visualizer screen starts appearing.
3. Navigate to 'WeatherLarge.Json' button by using Tab key and activate it.
4. WeatherLarge.Json adaptive card will appear on right sight of the pane.
5. Navigate to WeatherLarge.Json card by using Tab key.
6. Press the tab key and navigate to the 'Week Name' link and observe the issue.

**Actual Result:**
While navigating to the Wednesday Link, Content under tooltip is not visible properly. It is getting cropped.

**Expected Result:**
Tooltip for the days link should be visible properly. High and Low value should be appear properly.

**User Impact:**
It will impact the user as they will not be able to see the entire information contain under the tooltip due to this there might be a chance that user skip the important information.

Have feedback to share on Bugs ? please tag bug as “A11yRCA” and add your feedback in the comment section.

### Screenshots

![In normal mode, Tooltip of Days name is getting cropped](https://github.com/user-attachments/assets/9d58d13e-8e58-4141-abe9-020309054f81)

### Card JSON

```JSON
NA
```
